lib/src/base-styles/_nx-page-layout.scss

```diff
@@ -16,27 +16,35 @@
   background-color: $nx-site-background-color;
   height: 100%;
   margin: 0;
-  // prevents scrollbar flicker as tooltips appear in IE11
-  overflow-y: hidden;
   padding: 0;
 }
 
 .nx-page {
-  @include container-vertical;
-
-  align-items: center;
-  display: flex;
-  flex-direction: column;
-  height: 100%;
+  display: grid;
   min-width: var(--page-width-min);
+  height: 100%;
+  width: 100%;
+
+  // NOTE: Most of this template is only used when using nx-page-content--full-width (or no nx-page-content at all).
+  // In traditional layouts that use plain nx-page-content, that element manages everything below the second
+  // row of this template.
+  grid-template:
+    'old-header     old-header  old-header' auto
+    'global-sidebar   notices      notices' auto
+    'global-sidebar   header        header' auto
+    'global-sidebar old-sidebar    content' 1fr /
+      auto             auto            1fr;
 }
 
 .nx-page-content {
   @include container-horizontal;
 
+  justify-self: center;
   box-sizing: border-box;
   display: flex;
-  flex-grow: 1;
+
+  // See the comment above on nx-page's grid template
+  grid-area: 3 / 1 / -1 / -1;
```

Comment:
I think that this could use further explanation.

Reply:
Why doesn't this start at row 2?

Reply:
> Why doesn't this start at row 2?

This way the system notices are supported in the old layout as well

Reply:
> I think that this could use further explanation.

c66781d5

Reply:
> > Why doesn't this start at row 2?
> 
> This way the system notices are supported in the old layout as well

This puts us in a weird place though, we're allocating space for a notice that RSC doesn't support, for which we have no designs, and for which we don't even have a ticket in the backlog.

Reply:
> This puts us in a weird place though, we're allocating space for a notice that RSC doesn't support, for which we have no designs, and for which we don't even have a ticket in the backlog.

One of the followup PRs that I have linked in the description is to document the existence of this slot.  For the time being it would be up to products what they put in the slot, so there isn't really a need for mockups at that level.  However, @dturner32 has also just created RSC-570 which will make RSC components (or at least guidance on usage of existing components) in this slot.  I assume there will be mockups for that ticket.

The reason I felt the need to do this now is because I didn't want downstream projects that need a places for system notices in the short term to have to completely rewrite our carefully-crafted grid template in order to fit them in.

Reply:
> > This puts us in a weird place though, we're allocating space for a notice that RSC doesn't support, for which we have no designs, and for which we don't even have a ticket in the backlog.
> 
> One of the followup PRs that I have linked in the description is to document the existence of this slot. For the time being it would be up to products what they put in the slot, so there isn't really a need for mockups at that level. However, @dturner32 has also just created RSC-570 which will make RSC components (or at least guidance on usage of existing components) in this slot. I assume there will be mockups for that ticket.
> 
> The reason I felt the need to do this now is because I didn't want downstream projects that need a places for system notices in the short term to have to completely rewrite our carefully-crafted grid template in order to fit them in.

The existence of the tickets makes me happy